addons/account/wizard/account_invoice_import.py

```diff
@@ -16,7 +16,7 @@ def _get_default_journal_id(self):
 
     @api.multi
     def _create_invoice_from_file(self, attachment):
-        self = self.with_context(default_journal_id=self.journal_id.id)
+        self = self.with_context(default_journal_id=(self.journal_id or self._get_default_journal_id()).id)
```

Comment:
is the wizard still used? From what I can see, it does not seems to be the case, should we not delete it (and the view associated) and move the method to create the invoice elsewhere? (this method could be rewritten in a single method)
l10n_it_edi is overwritting this method so you might have to look out for it too.